src/SFA.DAS.RoATPService.Application.Api/Controllers/UpdateOrganisationController.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace SFA.DAS.RoATPService.Application.Api.Controllers
+{
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Mvc;
+    using SFA.DAS.RoATPService.Application.Api.Middleware;
+    using Swashbuckle.AspNetCore.SwaggerGen;
+    using System.Collections.Generic;
+    using System.Net;
+    using MediatR;
+    using Microsoft.Extensions.Logging;
+    using RoATPService.Api.Types.Models;
+    using System.Threading.Tasks;
+
+    [Authorize(Roles = "RoATPServiceInternalAPI")]
+    [Route("api/v1/[controller]")]
+    public class UpdateOrganisationController : Controller
+    {
+        private readonly ILogger<UpdateOrganisationController> _logger;
+        private readonly IMediator _mediator;
+
+        public UpdateOrganisationController(ILogger<UpdateOrganisationController> logger, IMediator mediator)
+        {
+            _logger = logger;
+            _mediator = mediator;
+        }
+
+        [HttpPost]
```

Comment:
Dave, I was thinking about this... I think this should be HttpPut.  Let's discuss in Slack...

Reply:
Yeah makes sense